src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newSingleThreadExecutor(); // Provide implementation that fits your needs
```

Comment:
It's a good practice to shut down the ExecutorService. Consider adding a try-finally block to ensure 'executor.shutdown()' is called even if an exception occurs.